frame/staking/src/benchmarking.rs

```diff
@@ -29,6 +29,14 @@ const MAX_SPANS: u32 = 100;
 const MAX_VALIDATORS: u32 = 1000;
 const MAX_SLASHES: u32 = 1000;
 
+macro_rules! do_whitelist {
+	($acc:ident) => {
+		frame_benchmarking::benchmarking::add_to_whitelist(
+			frame_system::Account::<T>::hashed_key_for(&$acc).into()
+		);
+	}
+}
```

Comment:
at some point you could use a simple function:
```rust
fn whitelist_account<T: Trait>(acc: &T::AccountId) {
	frame_benchmarking::benchmarking::add_to_whitelist(
		frame_system::Account::<T>::hashed_key_for(acc).into()
	);
}
```

Reply:
yeah I am using the same in another PR as well, later on I will refactor them into `frame/support/benchmarking` 🙈